e2e/repo-page.spec.ts

```diff
@@ -13,7 +13,7 @@ test("Loads a repository page", async ({ page }) => {
 
 test("repository page has an OG image", async ({ page }) => {
   await page.goto("/s/open-sauced/app");
-  const expectedUrl = `${config.use?.baseURL}/og-images/repository/open-sauced/app/30?description=%F0%9F%8D%95+The+path+to+your+next+contribution.`;
+  const expectedUrl = `${config.use?.baseURL}/og-images/repository/open-sauced/app/30?description=%F0%9F%8D%95+Insights+into+your+entire+open+source+ecosystem.`;
```

Comment:
Not related to the PR, but it looks like we changed the description of our repo, so it caused the E2E test to fail. I've updated it to replact the new open-sauced/app description.